lib/perl/Genome/Model/Tools/CopyNumber/CnView.pm

```diff
@@ -353,6 +347,26 @@ sub execute{
   return(1);
 }
 
+sub get_ideogram {
+  my $self = shift;
+  my $user_reference_build = shift;
+  my $clinseq_annotations_dir = shift;
+  my $default_build36 = Genome::Model::Build->get(101947881);
+  my $default_build37 = Genome::Model::Build->get(106942997);
+  my $default_build38 = Genome::Model::Build->get("4ec1c5bd1f6941b8a99f2e230217cb91");
+  my $ideogram_map = {
+    "36" => {"build" => $default_build36, "ideogram" => $clinseq_annotations_dir . "/hg18/ideogram/ChrBandIdeogram.tsv"},
+    "37" => {"build" => $default_build37, "ideogram" => $clinseq_annotations_dir . "/hg19/ideogram/ChrBandIdeogram.tsv"},
+    "38" => {"build" => $default_build38, "ideogram" => $clinseq_annotations_dir . "/hg19/ideogram/ChrBandIdeogram.tsv"}
```

Comment:
I'm surprised this path is identical between builds 37 and 38.  That seems like it could lead to accidentally mixing them up!  It does match `build38-20160302`, though.